core/src/com/unciv/logic/multiplayer/storage/DropBox.kt

```diff
@@ -169,7 +151,6 @@ object DropBox: FileStorage {
 
     @Suppress("PropertyName")
     private class MetaData: FileMetaData {
-        var name = ""
```

Comment:
One example for `@Suppress("unused") // why` being the better choice... Another line I already itched to add the `// why` to the name suppress, but it never fit the current branch I was working on.

Reply:
> even though this is currently unused, it should stay so that the response stays properly described

If it is a matter of documenting/describing the code, I'd suggest using the comments. Having the property, class and no use of it - just wasting of .jar size and its performance. You can argue that a few bytes in memory does not change the picture much, but that's a matter of systematic approach: droplets make the ocean.